tests/app/migrations/0004_customvideomodel_error_message.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.13 on 2024-05-31 17:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("app", "0003_alter_customvideotrack_file_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="customvideomodel",
+            name="error_message",
```

Comment:
i think we no longer need this?

Reply:
We've added error message previously to the abstract `AbstractVideo` model. So we need this migration for the test app.

Reply:
i don't understand why error_message is needed?